app/javascript/components/visualization/controls/ScatterPlotLegend.jsx

```diff
@@ -194,6 +193,9 @@ function getShowHideEnabled(hiddenTraces, countsByLabel) {
  * Get color for the label, which can be applied to e.g. the icon or the trace
  */
 function getColorForLabel(label, customColors={}, editedCustomColors={}, i) {
+  if (label === '--Unspecified--' && !editedCustomColors[label] && !customColors[label]) {
+    return 'rgba(80, 80, 80, 0.4)'
```

Comment:
This has a [1.96 contrast ratio](https://www.joedolson.com/tools/color-contrast.php?type=rgb&color=80%2C80%2C80&color2=255%2C255%2C255&alpha=.4), which is inaccessible for some users.  Bumping alpha to 0.8 makes it [accessible](https://www.joedolson.com/tools/color-contrast.php?type=rgb&color=80%2C80%2C80&color2=255%2C255%2C255&alpha=.8).  I can imagine an argument that "Unspecified" cells are of minimal interest and peripheral to the graphic, so I don't necessarily consider this a blocker.

```suggestion
    return 'rgba(80, 80, 80, 0.8)'
```

Reply:
Yeah, that's a good point about accessibility.  My sense is that you're right, we're trying to make it clear that these points are peripheral, and so making them *very* different from the specified points is actually a net benefit to visually impaired users, even if it means they might not see them at all.